source/MaterialXCore/Node.h

```diff
@@ -269,7 +271,7 @@ class GraphElement : public InterfaceElement
 
     /// Flatten any references to graph-based node definitions within this
     /// node graph, replacing each reference with the equivalent node network.
-    void flattenSubgraphs(const string& target = EMPTY_STRING);
+    void flattenSubgraphs(const string& target = EMPTY_STRING, FlattGraphPredicate=nullptr);
```

Comment:
I'd recommend providing the variable name ("filter") in this declaration for clarity.